### DISCUSSION TITLE
[Help Documents] Dockerizing the Theia Binary

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#13453] [Q&A] [answered]
Hi all,
I am trying to create a theia docker binary to integrate with Eclipse Che/Theia Cloud. However, I am unable to find any proper documentation. Do we have any ?

Thanks

**Comment by JonasHelming** [comment, marked as answer]
Hi,

probably this is a good start: https://github.com/eclipse-theia/theia-blueprint?tab=readme-ov-file#docker-build
I am not aware of any dedicated documentation, but from the example above, you can hopefully derive and adapt the necessary steps.

Best regards,
Jonas

**Comment by maheshrajrp** [comment]
Thanks @JonasHelming .  This should help.
